core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/codesystem/version/CodeSystemVersionCreateRequest.java

```diff
@@ -221,13 +221,6 @@ private void acquireLocks(ServiceProvider context, String user, Collection<CodeS
 
 				lockTargetsByContext.put(lockContext, lockTarget);
 			}
-			
-		} catch (final OperationLockException e) {
```

Comment:
Is this handled gracefully somewhere else?

Reply:
Hmm, this needs to be converted to `LockedException`, I'll add something to make that easier. Or we could throw that exception instead of any low-level ones.